tensorflow/python/distribute/mirrored_strategy.py

```diff
@@ -388,13 +361,15 @@ def _initialize_multi_worker(self, devices):
 
     logging.info("Using MirroredStrategy with remote devices %r", devices)
 
-  def _input_workers_with_options(self, options=None):
+  def _input_workers_with_options(self, options=None, input_workers_devices=None):
+    if not self._input_workers_devices:
```

Comment:
i think you want to check `if not input_workers_devices` here instead?

Reply:
modified to check for input_workers_devices